XF.Material/XF.Material.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageId>XF.Material</PackageId>
     <Description>A Xamarin.Forms library for Xamarin.Android and Xamarin.iOS to implement Google's Material Design.</Description>
     <DisableFastUpToDateCheck>true</DisableFastUpToDateCheck>
+    <Version>1.7.7.1</Version>
```

Comment:
remove this, as it may be set by a batch ?

Reply:
This is my first pull request. So I am newbie. All comment is for me or @softlion? Thank you for your interest.

Reply:
It’s cool you made a pr

Reply:
Thank you :)

Reply:
Hello will you get this pr? @softlion  @martijn00